sdk/test/Services/RestXmlProtocol/Generated/Model/Internal/MarshallTransformations/XmlAttributesRequestMarshaller.cs

```diff
@@ -64,7 +64,7 @@ public IRequest Marshall(XmlAttributesRequest publicRequest)
             {   
                 xmlWriter.WriteStartElement("XmlAttributesRequest", "");
                 if(publicRequest.IsSetAttr())
-                    xmlWriter.WriteAttributeString("test", "",StringUtils.FromString(publicRequest.Attr));
+                        xmlWriter.WriteAttributeString("","test", "",StringUtils.FromString(publicRequest.Attr));
```

Comment:
Is the extra space before `xmlWriter` intentional? If you're making a change, it'd also be good to add spaces between the `,`, e.g.: 
```csharp
xmlWriter.WriteAttributeString("", "test", "", StringUtils.FromString(publicRequest.Attr))
```

Reply:
ah good catch, removed the extra indentation